lib/index.js

```diff
@@ -85,10 +85,25 @@ var bootStrap = function (options) {
 
     refreshHandle(com, win);
 
-    eventListener.add(global, 'load', function onloadDelayedHandler() {
+    function resizeHandler(name) {
         var size = childrenSize(gardr.container);
-        pluginApi.trigger('banner:rendered', size);
+        pluginApi.trigger('banner:' + name, size);
         com.rendered(size);
+    }
+
+    eventListener.add(doc, 'readystatechange', function readyStateChangeHandler() {
+        if (/interactive|complete/.test(doc.readyState)) {
+            resizeHandler(doc.readyState);
```

Comment:
`complete` is the same as load, and `interactive is the same as `DOMContentLoaded` (even though IE might behave differently). So do you actually need readystatechange at all?